### PR TITLE
Build alpine 3.20 backports

### PR DESCRIPTION
**Pull request by at-wat**
- Temporary excluded colcon packages due to the build error (https://github.com/seqsense/aports-ros-experimental/pull/928)
- Remove packages provided by official Alpine repo
  - assimp
  - ffcall
  - libao
  - libsigsegv
  - netpbm
  - py3-importlib-resources
  - py3-mongo
  - py3-pytest-repeat
  - py3-qt5
  - py3-rerunfailures
  - py3-wxpython
- Remove deprecated packages
  - py3-rosinstall
  - py3-vcstools
  - py3-wstool
- Update package versions

### CODE REVIEW

File: build-repo.sh [hunk 1]
```diff
@@ -86,7 +78,7 @@ then
   ); do
     echo "Resigning ${index}"
     rm -f ${index}
-    apk index -o ${index} $(find $(dirname ${index})/../ -name '*.apk')
+    apk index --allow-untrusted -o ${index} $(find $(dirname ${index})/../ -name '*.apk')
```
Comment (f-fl0) [inline build-repo.sh:81]: Why do we need to add `--allow-untrusted` now?

File: Makefile [hunk 2]
```diff
@@ -66,10 +66,11 @@ all:
...
 UPDATE_TARGETS ?=
+export UPDATE_TARGETS
 
 .PHONY: update-checksum
 update-checksum:
 	docker run --rm \
 		-v $(CURDIR):/src \
 		--entrypoint /update-checksum.sh \
-		$(BUILDER_NAME):$(BUILDER_TAG) $(UPDATE_TARGETS)
+		$(BUILDER_NAME):$(BUILDER_TAG) $${UPDATE_TARGETS}
```
Comment (f-fl0) [inline Makefile:76]: Was there a problem with the previous syntax?

File: build-repo.sh (hunk 1 above)
Comment (at-wat) [reply build-repo.sh:81]: `apk index` now validates that the apks are signed, but it's not yet signed as it's resigning

**Review by f-fl0 — APPROVED** [review]
LGTM

**Review by Taka-Kazu — APPROVED** [review]

**Pull request merged**